lib/commands.js

```diff
@@ -852,7 +852,8 @@ commands.waitFor = function(){
   }
 
   // default
-  opts.timeout = opts.timeout || 1000;
+  opts.timeout = opts.timeout || config.getDOMConfig().timeout;
+
   opts.pollFreq = opts.pollFreq || 200;
```

Comment:
Maybe we could add also a configuration for polling frequency? However I don't remember that we configure this settings at cbt 🤔 @ysukhoverkhov what do you say on this?